internal/util/argParser.go

```diff
@@ -127,7 +127,7 @@ func mapArgsToInput(args []string) []Input {
 			inputList = append(inputList, input)
 			input = Input{}
 		} else {
-			input.Value = arg
+			input.Value = NormalizeName(arg)
```

Comment:
Actually now that I think about it, I'm not sure that this (and APPX-89) is correct. While it might b e true that `Spec.Parameters.Maintenance.DayOfWeek` must be lowercase, it might be that other fields must be mixed case, no?

For example, I think fields like `imagePullPolicy` MUST be `Always`, and `always` is invalid.

Reply:
True, some consistency would have been amazing, oh well.